From aed9c475d6c6017bf170d616cd9daab65baea5d2 Mon Sep 17 00:00:00 2001
From: Peter Swain <pswain@cavium.com>
Date: Mon, 26 Nov 2018 06:05:38 +0300
Subject: [PATCH 0757/1051] mtd: nand_ids: remove obsolete flags from mt29f
 device

This was introduced for a Micron MT29F1G01AAADD, supported by
staging/mt29f_spinand, which never needed this obsolete flag

Signed-off-by: Peter Swain <pswain@cavium.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/mtd/nand/raw/nand_ids.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mtd/nand/raw/nand_ids.c b/drivers/mtd/nand/raw/nand_ids.c
index f604a4db8ae5..e9e20cca9011 100644
--- a/drivers/mtd/nand/raw/nand_ids.c
+++ b/drivers/mtd/nand/raw/nand_ids.c
@@ -172,7 +172,7 @@ struct nand_flash_dev nand_flash_ids[] = {
 		.pagesize = 2048,
 		.chipsize = 128,
 		.erasesize = SZ_128K,
-		.options = NAND_BBT_SCAN2NDPAGE | NAND_ECC_CUSTOM_PAGE_ACCESS,
+		.options = NAND_BBT_SCAN2NDPAGE,
 		.id_len = 2,
 		.oobsize = 64,
 		.ecc = NAND_ECC_INFO(4, 512),
-- 
2.17.1

